html/ops/generate_account_ownership_keys.php

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env php
+<?php
+
+// This file is part of BOINC.
+// http://boinc.berkeley.edu
+// Copyright (C) 2008 University of California
+//
+// BOINC is free software; you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// BOINC is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
+
+require_once("../inc/boinc_db.inc");
+require_once("../inc/user.inc");
+require_once("../inc/util.inc");
+
+$config = get_config();
+$keydir = parse_config($config, "<key_dir>");
+$private_key_path = "file://$keydir/account_ownership_private.pem";
+$public_key_path = "file://$keydir/account_ownership_public.pem";
+
```

Comment:
These four ines are duplicated with other code and should be consolidated to a single location (maybe create inc/account_ownership.inc?)